doc/source/programs/gdal_raster_pipeline.rst

```diff
@@ -94,6 +94,18 @@ Details for options can be found in :ref:`gdal_raster_edit_subcommand`.
 
 Details for options can be found in :ref:`gdal_raster_reproject_subcommand`.
 
+* resize [OPTIONS]
+
+.. code-block::
+
+    Resize a raster dataset.
+
+    Options:
+      --size <width>,<height>                              Target size in pixels [required]
+      -r, --resampling <RESAMPLING>                        Resampling method. RESAMPLING=nearest|bilinear|cubic|cubicspline|lanczos|average|mode
+
```

Comment:
Should we mention what is the default? I guess that nearest is the default.

Reply:
good point. default added